tests/analysis/test_tables.py

```diff
@@ -0,0 +1,98 @@
+"""Test table generators."""
+from collections.abc import Callable
+from unittest.mock import call
+
+import pandas as pd
+import pytest
+from pytest_mock import MockerFixture
+
+from coin_test.analysis.tables import MetricsGenerator, SummaryTearSheet, TearSheet
+from coin_test.backtest import BacktestResults
+
+
+@pytest.fixture
+def mock_metrics() -> list[str]:
+    """Mock metrics."""
+    return [
+        "Average Annual Return (%)",
+        "Sharpe Ratio",
+        "Sortino Rato",
+        "Max Drawdown (%)",
+        "Calmar Ratio",
+    ]
+
+
+def test_metrics_generator_single_metrics(backtest_results: BacktestResults) -> None:
+    """Successfully get metrics for a single backtest."""
+    metrics = MetricsGenerator._single_metrics(backtest_results)
+
+    assert (metrics["Sharpe Ratio"] - 3.817) < 1e-3
```

Comment:
can we be more standard about this comparison and use `pytest.almost_equal` https://stackoverflow.com/questions/8560131/pytest-assert-almost-equal